### PR TITLE
N20: Bevy Engine

### PR DESCRIPTION
**Pull request by cart**
Sorry for missing the soft deadline. I focused too much on the Bevy release yesterday and let this slip. Hope we're not too late!

_Part of #351_

### CODE REVIEW

File: content/posts/newsletter-020/index.md [hunk 1]
```diff
@@ -988,6 +988,45 @@ can get glimpses of its development by following the author on [Twitter][h3r2tic
...
+- Rich text: style text "spans" with different colors / fonts while still
+respecting layout
+- HIDPI text: render crisp text at any resolution
+- 2D world space text, world to screen space conversions, 2d/3d orthographic
+camera improvements, render layers, sprite flipping, improved color space handling,
+wireframes, timer improvements, and more!
+
+_Discussions:
+[/r/rust](https://www.reddit.com/r/rust/comments/mljg39/bevy_05/),
+[hacker news](https://news.ycombinator.com/item?id=26716166)
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-020/index.md:1023]:
```suggestion
[hacker news](https://news.ycombinator.com/item?id=26716166),
```

**Review by 17cupsofcoffee — APPROVED** [review]

**Review by ozkriff — APPROVED** [review]
It's a little bit longer than 1000 chars but I guess it's ok in this case

**Comment by ozkriff** [comment]
Merged, thanks for the PR!

**Comment by cart** [comment]
> It's a little bit longer than 1000 chars but I guess it's ok in this case

Thanks for the heads up. I'll consolidate even more next time

**Pull request merged**